examples/master.js

```diff
@@ -135,6 +136,26 @@ async function startMaster(localView, remoteView, formValues, onStatsReport, onR
                 secretAccessKey: formValues.secretAccessKey,
                 sessionToken: formValues.sessionToken,
             },
+            requestSigner: {
+                getSignedURL: async function(signalingEndpoint, queryParams, date) {
+                    const signer = new KVSWebRTC.SigV4RequestSigner(formValues.region, {
```

Comment:
Don't we have to remove this call from the SignalingClient if we add it here? Additionally, I'm not sure if we should get rid of that abstraction just to calculate certain metrics.

Reply:
Not sure what you mean, we are allowed to override this function. If not provided, we will use the default one.

https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-js/blob/de209884fcd388c3fea4937a5b99354ef9519af3/src/SignalingClient.ts#L101-L106